user/deployment/google-app-engine.md

```diff
@@ -15,9 +15,9 @@ deploy:
   project: "YOUR PROJECT ID"
 ```
 
-You can create a Service Account by going to the [Google Cloud Console](http://console.developers.google.com), go to "API Manager" -> "Credentials",
-then click "Add Credential" and "Service account key", finally clicking "JSON" to download the JSON key.
-Also, you need to enable "Google App Engine Admin API" from the "Dashboard" or "Library" sections.
+Download your Service Account JSON file from the [Google Cloud Console](http://console.developers.google.com):
```

Comment:
Maybe this line could be deleted because the following line seems enough (after adding "your Service Account JSON file" to the end)